### PR TITLE
fix: 修复行头为空无默认角头指标文字问题

### PR DESCRIPTION
**Pull request by xingwanying**
### 👀 PR includes

🐛 Bugfix

- [x] Solve the issue and close #0

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌ image | ✅  image|

### CODE REVIEW

File: packages/s2-core/src/cell/corner-cell.ts [hunk 1]
```diff
@@ -73,18 +74,19 @@ export class CornerCell extends HeaderCell {
...
-    }
-
     const { x } = this.getContentArea();
     const { y, height } = this.getCellArea();
 
     const textStyle = this.getTextStyle();
-    const { formattedValue } = this.getFormattedFieldValue();
-    const cornerText = this.spreadsheet.options?.cornerText || formattedValue;
 
+    let cornerText: string;
```
Comment (lijinke666) [inline packages/s2-core/src/cell/corner-cell.ts:82]: 单独抽个 getCornerText 的函数把

File: packages/s2-core/src/cell/corner-cell.ts [hunk 2]
```diff
@@ -368,4 +364,15 @@ export class CornerCell extends HeaderCell {
       y: 0,
     };
   }
+
+  protected getCornerText(label: string) {
```
Comment (lijinke666) [inline packages/s2-core/src/cell/corner-cell.ts:368]:
```ts
 protected getCornerText(label: string): string {
    if (isEqual(label, EXTRA_FIELD)) {
      return this.spreadsheet.options?.cornerText || DEFAULT_CORNER_TEXT;
    }

    const { formattedValue } = this.getFormattedFieldValue();
    return formattedValue;
  }
```

**Review by wjgogogo — APPROVED** [review]

**Pull request merged**